x/language.rb

```diff
@@ -0,0 +1,15 @@
+module X
+  class Language
+    def self.of(id)
+      tracks.fetch(id.to_s) do
+        @@tracks[id.to_s] = Track.find(id).language
```

Comment:
I tried making some tests for the cache but I'm having problems in doing so. Since we now use the cache, I'm getting inconsistent tests since the cache is a class variable. For example, when running the tests, it sometimes all passes since there is this test which runs first and populates the cache. On subsequent tests on the same run, this cache is used. Other runs fail.